mne/preprocessing/ica.py

```diff
@@ -2413,7 +2411,7 @@ def corrmap(icas, template, threshold="auto", label=None, ch_type="eeg",
         logger.info('Displaying selected ICs per subject.')
 
     for ii, (ica, max_corr) in enumerate(zip(icas, mx)):
-        if (label is not None) and (not hasattr(ica, 'labels_')):
+        if label is not None:
```

Comment:
You're right, my bad. Do we need this condition anyway? `ica.labels_` is always set in this branch, so no need to set it to an empty `dict`.